docs/docsite/rst/user_guide/playbooks_variables.rst

```diff
@@ -1030,6 +1030,8 @@ If multiple variables of the same name are defined in different places, they get
 
 Here is the order of precedence from least to greatest (the last listed variables winning prioritization):
 
+  * ansible.cfg (eg :ref:`ssh_executable <ANSIBLE_SSH_EXECUTABLE>`)
+  * environment variables (eg :envvar:`ANSIBLE_SSH_EXECUTABLE`)
```

Comment:
not variables, you are mixing variable precedence with the configuration resolution

Reply:
Removed.